pinot-spi/src/main/java/org/apache/pinot/spi/data/DateTimeGranularitySpec.java

```diff
@@ -53,38 +55,49 @@ public DateTimeGranularitySpec(String granularity) {
     char separator;
     int sizePosition;
     int timeUnitPosition;
+    int minTokens = NUM_TOKENS;
+    int maxTokens = NUM_TOKENS;
```

Comment:
(minor) Move the assignment into the if check for readability

Reply:
Done